Core/Core/Inbox/AttachmentPicker/View/AudioPickerView.swift

```diff
@@ -0,0 +1,346 @@
+                Button {
+                    viewModel.retakeButtonDidTap.accept(controller)
+                } label: {
+                    Text("Retake", bundle: .core)
+                        .foregroundStyle(Color.white)
+                }
+            }
+            .frame(maxWidth: .infinity)
+
+            VStack(alignment: .center) {
+                if (!viewModel.isPlaying) {
+                    Button {
+                        viewModel.startPlaying()
+                    } label: {
+                        Image.playSolid
+                            .foregroundStyle(Color.white)
+                    }
+                    .frame(width: 50, height: 50, alignment: .center)
+                    .accessibilityLabel(Text("Play audio recording", bundle: .core))
+                } else {
+                    Button {
+                        viewModel.pausePlaying()
+                    } label: {
+                        Image.pauseSolid
+                            .foregroundStyle(Color.white)
+                    }
+                    .frame(width: 50, height: 50, alignment: .center)
+                    .accessibilityLabel(Text("Pause audio recording", bundle: .core))
+                }
+            }
+            .frame(maxWidth: .infinity)
+
+            VStack(alignment: .trailing) {
+                Button {
+                    viewModel.useAudioButtonDidTap.accept(controller)
+                } label: {
+                    Text("Use Audio", bundle: .core)
+                        .foregroundStyle(Color.white)
+                }
+            }
+            .frame(maxWidth: .infinity)
+        }
+        .frame(maxWidth: .infinity)
+        .padding(.vertical, 12)
+        .padding(.horizontal, 12)
+        .background {
+            Color.black
+        }
+    }
+
+    private var recordControlView: some View {
+        HStack {
+            VStack(alignment: .leading) {
+                Button {
+                    viewModel.cancelButtonDidTap.accept(controller)
+                } label: {
+                    Text("Cancel", bundle: .core)
+                        .foregroundStyle(Color.white)
+                }
+            }
+            .frame(maxWidth: .infinity)
+            VStack(alignment: .center) {
+                if viewModel.isRecorderLoading {
+                    loadingIndicator
+                } else if (!viewModel.isRecording) {
+                    startRecordButton
+                } else {
+                    stopRecordingButton
+                }
+            }
+            .frame(maxWidth: .infinity)
+            VStack(alignment: .trailing) {
+            }
+            .frame(maxWidth: .infinity)
+        }
+        .frame(maxWidth: .infinity)
+        .padding(.vertical, 12)
+        .padding(.horizontal, 12)
+        .background {
+            Color.black
+        }
+    }
+
+    private var startRecordButton: some View {
+        Button {
+            viewModel.isRecorderLoading = true
```

Comment:
Maybe we could this to the `startRecording` function instead of directly changing the value here.